api/sbom.proto

```diff
@@ -53,6 +55,7 @@ message Node {
     repeated string file_types = 27; // File types
     map<int32,string> identifiers = 28;  // Software identifiers
     map<int32,string> hashes = 29;
+    Purpose primary_purpose = 30;
```

Comment:
In SPDX, the primary has a cardinality of one or more, so this should be a list:

```suggestion
    repeated Purpose primary_purpose = 30;
```

This obviously triggers changes in the dependent objects, sorry :/

Reply:
cool, no worries.  glad we have a chance to get this right.  before 0.2.0 is the time!